shell/platform/android/io/flutter/view/AccessibilityBridge.java

```diff
@@ -1306,7 +1306,7 @@ void updateSemantics(@NonNull ByteBuffer buffer, @NonNull String[] strings) {
                     event.setScrollX((int) position);
                     event.setMaxScrollX((int) max);
                 }
-                if (object.scrollChildren > 0) {
+                if (object.scrollChildren > 0 && object.childrenInHitTestOrder != null) {
```

Comment:
@jonahwilliams should know better but I suspect this is not going to be right.  We should not get a case where we have scroll children and we can't get them in hitTestOrder.

Reply:
Agreed. `object.scrollChildren >0 && object.childrenInHitTestOder == null` is an illegal state. We should fix whatever we're doing to get into this state rather than work around it.